hrm-service/package.json

```diff
@@ -1,20 +1,28 @@
 {
-  "name": "hrm",
+  "name": "@tech-matters/hrm-service",
   "version": "0.0.0",
-  "description": "",
+  "description": "The easiest way to run the tests locally is to run the test DB in docker.",
   "scripts": {
-    "build": "npx tsc",
-    "start": "node ./service-tests/db-init/migrate && cd ./dist && node ./bin/www",
+    "start": "run-s test:service:migrate start:run",
+    "start:run": "cd dist && node ./bin/www",
     "build-and-start": "npm run build && npm run start",
-    "test": "jest --passWithNoTests --forceExit --maxWorkers=1",
+    "docker:build": "docker build -t hrm-service -f Dockerfile ../",
+    "docker:compose:test:up": "docker compose -f ./docker-compose-test.yml up -d",
+    "docker:compose:test:down": "docker compose -f ./docker-compose-test.yml down",
+    "test": "run-s test:migrate test:unit test:service",
     "test:unit": "jest unit-tests",
-    "test:service": "node ./service-tests/db-init/migrate && jest --passWithNoTests --forceExit --maxWorkers=1 service-tests",
```

Comment:
This is a change that probably needs some discussion. I generally make the "main" test path as simple as possible. So it does spin up and tear down of required resources. I then provide subcommands to run less automated parts of the test for when a developer is actively working on tests and wants to spin up a persistent test env and then run the actual tests as quickly as possible to test changes.

This is a bit of a departure from the current test:service:local setup, but makes things a bit smoother for the multiple workspace npm run pattern we use at the top level.